Resources/Prototypes/GameRules/unknown_shuttles.yml

```diff
@@ -70,5 +74,22 @@
     weight: 2
     reoccurrenceDelay: 30
     duration: 1
+    maxOccurrences: 2 # should be the same as [copies] in shuttle_incoming_event.yml
   - type: LoadMapRule
     preloadedGrid: SyndieEvacPod
+
+- type: entity
+  id: UnknownShuttleNTQuark
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    startAnnouncement: null # It should be silent.
+    weight: .5 # Hopefully this is uncommon enough, it needs to be uncommon enough that people wont waste time metaknowledging it.
+    earliestStart: 45 # late to hopefully have enough ghosts to fill all roles quickly.
+    reoccurrenceDelay: 30
+    duration: 1
+    minimumPlayers: 25
+    maxOccurrences: 1 # should be the same as [copies] in shuttle_incoming_event.yml
+  - type: LoadMapRule
+    preloadedGrid: NTQuark
```

Comment:
Looks like a feature to me

Reply:
What if I'm fixing the lack of it existing problem? :godo:

Reply:
Also, I contend, its very cool.